k9mail/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -1616,4 +1616,9 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
             }
         }
     }
+
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
```

Comment:
But we could remove it if you are planning to build a Layout file for landscape mode in the future. (Changing the layout file will be handled automatically by the system).



I inserted this code because I have found that we are not building a Layout file for landscape mode , thus , we are not handling orientation change .According to [the documentation](https://developer.android.com/guide/topics/resources/runtime-changes.html#HandlingTheChange)